src/Microsoft.AspNet.Owin/OwinExtensions.cs

```diff
@@ -67,21 +68,32 @@ public static IApplicationBuilder UseOwin(this IApplicationBuilder builder, Acti
 
         public static IApplicationBuilder UseBuilder(this AddMiddleware app)
         {
+            return app.UseBuilder(configureServices: null);
+        }
+
+        public static IApplicationBuilder UseBuilder(this AddMiddleware app, Action<IServiceCollection> configureServices)
```

Comment:
Sure, it's still bad layering at the expense of one extra line of user code. Doesn't seem necessary.